backend/editor/entries.py

```diff
@@ -286,6 +286,57 @@ def is_valid_branch_name(self):
         """
         return parser_utils.normalize_text(self.branch_name, char="_") == self.branch_name
 
+    async def create_project(self, description):
+        """
+        Helper function to create a node with label "PROJECT"
+        """
+        query = """
+            CREATE (n:PROJECT)
+            SET n.id = $project_name
+            SET n.taxonomy_name = $taxonomy_name
+            SET n.branch_name = $branch_name
+            SET n.description = $description
+            SET n.status = $status
+            SET n.created_at = datetime()
+        """
+        params = {
+            "project_name": self.project_name,
+            "taxonomy_name": self.taxonomy_name,
+            "branch_name": self.branch_name,
+            "description": description,
+            "status": "LOADING",
+        }
+        await get_current_transaction().run(query, params)
+
+    async def get_project_info(self):
+        """
+        Helper function to get information about a Taxonomy Editor project
+        """
+        query = """
+            MATCH (p:PROJECT {id: $project_name})
+            RETURN p AS info
+        """
+        params = {"project_name": self.project_name}
+
+        result = await get_current_transaction().run(query, params)  # returns a coroutine
+        info_record = await result.single()
+        info = info_record["info"] if info_record else None
+
+        return info
```

Comment:
There is also a `get_project_info` controller in `project_controller.py` you will have to modify it to use it in `api.py`
Message me if you want to update the call in `api.py` and delete this function in `entries.py` 